Jenkinsfile

```diff
@@ -288,3 +286,11 @@ def generateStepAndCatchError(version){
 def cleanDir(path){
   powershell label: "Clean ${path}", script: "Remove-Item -Recurse -Force ${path}"
 }
+
+def buildAndTest() {
+  retry(2) {
+    sh script: './scripts/jenkins/before_install.sh', label: 'Install dependencies'
```

Comment:
would it better to retry these individually? otherwise a failure in build.sh will trigger before_install.sh again unnecessarily, won't it?